### PR TITLE
Symfonize planning/postes_cfg/groupes.php and groupes2.php

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

File: public/setup/atomicupdate/pathToGroups.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+$sql[]=" UPDATE `{$dbprefix}acces` SET `page`='/framework-group' WHERE `page` = 'planning/postes_cfg/groupes.php';";
+
+$sql[]=" DELETE FROM `{$dbprefix}acces` WHERE `page`= 'planning/postes_cfg/groupes2.php';";
```
[the remark this answers is not in the document]
Comment (sillydebs) [reply public/setup/atomicupdate/pathToGroups.php:5]: We would have this line twice in db_data if I do not delete it : $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Configuration des tableaux - Modif',22,'Configuration des tableaux','','Planning',140);";

File: public/setup/atomicupdate/pathToGroups.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$sql[]=" UPDATE `{$dbprefix}acces` SET `page`='' WHERE `page` = 'planning/postes_cfg/groupes.php' OR `page`= 'planning/postes_cfg/groupes2.php';";
```
Comment (alexarn) [inline public/setup/atomicupdate/pathToGroups.php:3]: Missing change for groupe.php

File: public/setup/db_data.php [hunk 3]
```diff
@@ -50,6 +50,7 @@
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Personnel - Suppression', 21, 'Gestion des agents', 'personnel/suppression.php', 'Agents', 70);";
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Personnel - Valid', 21, 'Gestion des agents', 'personnel/valid.php', 'Agents', 70);";
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Gestion du personnel', 21, 'Gestion des agents', '', 'Agents', 70);";
+$sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Configuration des tableaux', 22, 'Configuration des tableaux', '','Planning',140);";
```
Comment (alexarn) [inline public/setup/db_data.php:53]: Why did you add this line ?

**Review by alexarn — CHANGES_REQUESTED** [review]

File: public/setup/db_data.php (hunk 3 above)
Comment (sillydebs) [reply public/setup/db_data.php:53]: Earlier, I asked you about having twice the same line in db_data. That is why I added this line.

**Comment by alexarn** [comment]
Merged on master
